CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Braintree iOS SDK Release Notes
 
+## Unreleased
+* Fixed the nesting of two keys in the `getBaseParameters` dictionary in the BraintreePayPalNativecheckout module
```

Comment:
```suggestion
* BraintreePayPalNativeCheckout (BETA)
  * Fix `merchant_account_id` and `correlation_id` keys to be nested at the top level of the internal create order request
```

Reply:
Changes look good to me! Just a small comment on CHANGELOG styling to match our previous entries.